tim/myPref.jshintrc

```diff
@@ -0,0 +1,4 @@
+{
+  "node": true,
+  "predef": ["describe", "it", "before", "after"]
+}
```

Comment:
This is a really small file compared to what I'm used to. Check out a ton of the options you can set for jshint: https://github.com/jshint/jshint/blob/master/examples/.jshintrc.